aea/crypto/ledger_apis.py

```diff
@@ -162,12 +162,12 @@ def transfer(self, identifier: str, crypto_object: Crypto, destination_address:
                     logger.info("transaction validated - exiting")
```

Comment:
regarding the `print` statements above:
- `print("TX Hash: ", hex_value.hex())` should be `logger.info("TX Hash {}".format(hex_value.hex()))`
- ` print("connect_to ...)` should be removed. It is an interesting feature to provide the link to block explorers, but we're not there yet.